workers/generic-worker/helper_test.go

```diff
@@ -372,6 +372,7 @@ func GWTest(t *testing.T) *Test {
 			// These ports are not exposed outside of the host. However, in CI they must differ from those of the
 			// generic-worker instance running the test suite.
 			LiveLogPortBase:    30583,
+			MaxTaskRunTime:     86400,
```

Comment:
For tests, I don't think we need such a high value. It doesn't hurt, but perhaps a value like 60 might be better, so that if someone writes a test that needs to take longer than 60s to run, they would at least see they have escaped the bounds of existing tests.

Reply:
I made it 180s because there are several tests that set that as the `MaxRunTime`.

Reply:
Scratch that, I made it 300s because there is a test that sets that as the `MaxRunTime`.